packages/ra-ui-materialui/src/input/ReferenceArrayInput.js

```diff
@@ -10,130 +10,6 @@ import LinearProgress from '../layout/LinearProgress';
 import Labeled from '../input/Labeled';
 import ReferenceError from './ReferenceError';
 
-const sanitizeRestProps = ({
```

Comment:
I just moved the View part at the bottom of the file so that the jsDoc appears first. It's easier to read that way.